### PR TITLE
 api intergration of the certificate and changes for the certificate.

### PR DESCRIPTION
**Pull request by farhanp1502**

### CODE REVIEW

File: projects/project-player-library/src/lib/assets/i18n/en.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "NO_FILES":"No {{fileType}} found for this project",
     "DELETE_ATTACHMENT":"Are you sure you want to delete this attachment ?",
     "REMOVE":"Remove",
-    "CERTIFICATE":"CERTIFICATE",
+    "CERTIFICATE":"Certificate",
```
Comment (Vinod-V3) [inline projects/project-player-library/src/lib/assets/i18n/en.json:48]: Why this change?

File: projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.html
Comment (Vinod-V3) [inline projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.html:1]: if the certificate criteria is not met except heading and the message nothing else should be displayed( projects details and download button)

File: projects/project-player-library/src/lib/pages/project-details/project-details.component.ts [hunk 2]
```diff
@@ -12,8 +12,10 @@ panelOpenStateForResources = false;
 validationTexts!: string[];
 learningResources: any[] = [];
 ngOnInit(): void {
-  this.getCertificateCriteria();
   this.learningResources = this.projectDetails?.learningResources;
+  setTimeout(()=>{
```
Comment (Vinod-V3) [inline projects/project-player-library/src/lib/pages/project-details/project-details.component.ts:16]: Async await didn't work?
Try using ngOnChanges and check

File: projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.html
Comment (farhanp1502) [reply projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.html:1]: yes it is done already

File: projects/project-player-library/src/lib/pages/project-details/project-details.component.ts (hunk 2 above)
Comment (farhanp1502) [reply projects/project-player-library/src/lib/pages/project-details/project-details.component.ts:16]: implemented the ngOnChange method

File: projects/project-player-library/src/lib/assets/i18n/en.json (hunk 1 above)
Comment (farhanp1502) [reply projects/project-player-library/src/lib/assets/i18n/en.json:48]: first it was capital case as we have camalize case for the header component.

File: projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.html [hunk 3]
```diff
@@ -1,14 +1,11 @@
 
-<div class="">
-
-    <div class="flex items-center">
-        <div class="text-lg font-bold">{{ "CERTIFICATE" | translate }}</div>
-    </div>
-    <div *ngIf="projectDetails.certificate.eligible"  class="mt-4">
+<lib-header [title]="'CERTIFICATE'"></lib-header>
+<div >
```
Comment (Vinod-V3) [inline projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.html:3]: use @if, @else

Comment (farhanp1502) [reply projects/project-player-library/src/lib/pages/certificate-page/certificate-page.component.html:3]: done

**Review by Vinod-V3 — APPROVED** [review]
